02-household-queries/qt_summaries-gpt-3.5-turbo-instruct-k_4.json

```diff
@@ -0,0 +1,851 @@
+            },
+            "Are live-in attendants included in food stamp households?": {
+                "score_sum": 0.194722245249761,
+                "derived_questions": [
+                    "Do the roommates often cook and eat together in the shared kitchen?"
+                ],
+                "entire_card": "Are live-in attendants included in food stamp households?\nNo, roomers and live-in attendants are not included as part of the food stamp household even if they share and prepare meals with the client. Live-in attendants can apply as a separate HH for their own food stamp benefits",
+                "summary": "No, live-in attendants are not included in food stamp households, even if they share and prepare meals with the client. They can apply for their own food stamp benefits as a separate household."
+            },
+            "What if my client shares meals but wants to apply for food stamps on their own?": {
+                "score_sum": 0.18720370925870566,
+                "derived_questions": [
+                    "Does the client share a kitchen with his roommates?"
+                ],
+                "entire_card": "What if my client shares meals but wants to apply for food stamps on their own?\nIf a client expresses a desire to apply for SNAP separately from those they live and currently eat with, consider the following:\nIs the client sharing meals only temporarily? (due to immediate necessity or a temporary living situation)\nIs the client expressing a desire to use SNAP benefits only for themselves? (not other people they live with)\nIs the client/their household willing to provide a signed statement to the county office that their intention is to eat/prepare meals separately from others that they live with?\nIf any of the above are answered\nYES\nand the other household members are\nnot mandatory\nSNAP HH members, then the client can apply on their own. (Find the mandatory HH members here:\nWho are mandatory HH members for food stamps?\n)\nPLEASE NOTE\nWe should never lead or encourage a client to apply themselves disingenuously, nor should we suggest to a client that defining their SNAP household in one way versus another would be better for them. The purpose of this clarification is to acknowledge that SNAP applicants who actively express a desire to receive and use SNAP for themselves only can be applied in this manner. We should explain the rule and allow the client to apply how they desire.\nIf the processing office has concerns or uncertainties about the SNAP household, they may ask the applicant(s) to supply verification (like a signed statement).",
+                "summary": "It is possible for a client to apply for SNAP benefits separately from their roommates if they are willing to provide a signed statement stating their intention to eat and prepare meals separately from others they live with."
```

Comment:
**Relevance to the question**: 🟢 Directly relevant

**Correctness as a summary of the Guru card**: 🟢 100% correct

**Helpfulness**: 🟢 Helpful

👏